packages/odyssey-react-labs/package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "@mui/material": "^5.10.12",
     "@mui/x-date-pickers": "^5.0.4",
+    "@okta/odyssey-react-mui": "^0.16.1",
```

Comment:
can it be a peer dependency with *

Reply:
https://yarnpkg.com/features/workspaces#workspace-ranges-workspace

![image](https://user-images.githubusercontent.com/86249047/207909225-2b3f272c-4923-4e75-87bd-0bcd2badf2e4.png)

Reply:
I'd love to change them all to `workspace:*`, but that's outta scope for this PR. We need to do that in a separate PR and have a ticket setup for it since I'm not sure if anything will break when we do.

When I was working on the plug 'n play transition, `workspace:*` was one of the things I made sure to change in my brand and didn't notice any issues locally related to it.

Reply:
407875bea5f6e2265a7144d2443f69d2fd844c92